xml/soc-operations-troubleshooting.xml

```diff
@@ -0,0 +1,85 @@
+<?xml version="1.0"?>
+<?xml-stylesheet href="urn:x-suse:xslt:profiling:docbook51-profile.xsl"
+ type="text/xml"
+ title="Profiling step"?>
+<!DOCTYPE chapter [
+ <!ENTITY % entities SYSTEM "entity-decl.ent"> %entities;
+]>
+
+<chapter xmlns="http://docbook.org/ns/docbook" xmlns:xi="http://www.w3.org/2001/XInclude" xmlns:xlink="http://www.w3.org/1999/xlink" version="5.0" xml:id="troubleshooting">
+ <title>Troubleshooting</title>
+ <para>
+  Before contacting support to help you with a problem on &cloud;, we
+  strongly recommended that you gather as much information about your system
+  and the problem as possible. For this purpose, &productname; ships with a
+  tool called <command>supportconfig</command>. It gathers system information
+  such as the current kernel version being used, the hardware, RPM database,
+  partitions, and other items. <command>supportconfig</command> also collects
+  the most important log files, making it easier for the supporters to
+  identify and solve your problem.
+ </para>
+ <section xml:id="shard-failures">
+  <title>Shard Failures</title>
+  <para>
+   This problem may occur the first time the Log Management Window is opened.
+   It occurs because the Elasticsearch index for the Monasca project does not
+   yet exist. A browser reload should fix the problem in most cases.
+  </para>
+  <informalfigure>
+   <mediaobject>
+    <imageobject>
+     <imagedata fileref="kibana_shard_failure-2.jpg" width="75%" format="JPG"/>
+    </imageobject>
+    <imageobject role="html">
+     <imagedata fileref="kibana_shard_failure-2.jpg" width="100%" format="JPG"/>
+    </imageobject>
+    <textobject><phrase>Kibana shard failure</phrase>
+    </textobject>
+   </mediaobject>
+  </informalfigure>
+ </section>
+ <section xml:id="something-went-wrong">
+  <title>Oops! Looks like something went wrong</title>
```

Comment:
Just as a general note to self - this should be renamed because it looks like there's a book build error ;)

Reply:
No, I'd assume you can't rename this. This is probably the actual error message you get from the software.